code/modules/organs/subtypes/xenos.dm

```diff
@@ -29,6 +29,27 @@
 	var/stored_plasma = 0
 	var/max_plasma = 500
 
+	organ_verbs = list(
+		/mob/living/carbon/human/proc/transfer_plasma
+		)
+
+/obj/item/organ/internal/xenos/plasmavessel/handle_organ_proc_special()
+	if(!istype(owner))
+		return
+
+	var/modifier = 1 - 0.5 * is_bruised()
+
+	if(owner.bloodstr.has_reagent("phoron"))
+		adjust_plasma(round(2 * modifier))
+
+	if(owner.ingested.has_reagent("phoron"))
+		adjust_plasma(round(4 * modifier))
```

Comment:
Why is this higher than the bloodstream one? Since this isn't part of phoron ticking, or seemingly related to human reagent containers beyond looking for phoron, this should probably be half of what the bloodstr offers, because it'll be hanging around longer.